src/services/task1.ts

```diff
@@ -21,11 +23,44 @@ export const task1Api = createApi({
         body: file,
       }),
     }),
+    changeFile: builder.mutation<
+      void,
+      Pick<IFile, "filename"> & Partial<IFile>
+    >({
+      query: ({ filename, content }) => ({
+        url: `/api/v1/files/${filename}`,
+        method: "PUT",
+        body: { content },
+      }),
+      async onQueryStarted(
```

Comment:
In real project you usually invalidate (refetch) data from server instead of updating it locally. Check https://redux-toolkit.js.org/rtk-query/api/createApi#invalidatestags and https://redux-toolkit.js.org/rtk-query/usage/automated-refetching